src/styles/blocks/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  width: #{$card-width};
+  max-height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 0 16px;
+
+  &:nth-child(-n+4) {
+    margin-bottom: 46px;
```

Comment:
You have to have `.catalog__card` element and all margins you will use for this element.